src/modules/iasql_functions/rpcs/iasql_get_sql_since.ts

```diff
@@ -0,0 +1,290 @@
+  orm: TypeormWrapper,
+): Promise<string[]> {
+  const queries: string[] = [];
+  for (const cl of changeLogs) {
+    let query: string = '';
+    switch (cl.changeType) {
+      case AuditLogChangeType.INSERT:
+        const valuesToInsert = await Promise.all(
+          Object.entries(cl.change?.change ?? {})
+            .filter(([k, v]: [string, any]) => k !== 'id' && v !== null)
+            .map(async ([k, v]: [string, any]) => await getValue(cl.tableName, k, v, mbt, orm)),
+        );
+        query = `
+          INSERT INTO ${cl.tableName} (${Object.keys(cl.change?.change ?? {})
+          .filter((k: string) => k !== 'id' && cl.change?.change[k] !== null)
+          .join(', ')})
+          VALUES (${valuesToInsert.join(', ')});
+        `;
+        break;
+      case AuditLogChangeType.DELETE:
+        const valuesToDelete = await Promise.all(
+          Object.entries(cl.change?.original ?? {})
+            .filter(([k, v]: [string, any]) => k !== 'id' && v !== null)
+            .map(async ([k, v]: [string, any]) => await getValue(cl.tableName, k, v, mbt, orm)),
+        );
+        query = `
+          DELETE FROM ${cl.tableName}
+          WHERE ${Object.entries(cl.change?.original ?? {})
+            .filter(([k, v]: [string, any]) => k !== 'id' && v !== null)
+            .map(([k, _]: [string, any], i) => `${k} = ${valuesToDelete[i]}`)
+            .join(' AND ')};
+        `;
+        break;
+      case AuditLogChangeType.UPDATE:
+        const updatedValues = await Promise.all(
+          Object.entries(cl.change?.change ?? {})
+            .filter(([k, _]: [string, any]) => k !== 'id')
+            .map(async ([k, v]: [string, any]) => await getValue(cl.tableName, k, v, mbt, orm)),
+        );
+        const conditionValues = await Promise.all(
+          Object.entries(cl.change?.original ?? {})
+            .filter(([k, _]: [string, any]) => k !== 'ami' && k !== 'id')
+            .map(async ([k, v]: [string, any]) => await getValue(cl.tableName, k, v, mbt, orm)),
+        );
+        query = `
+          UPDATE ${cl.tableName}
+          SET ${Object.entries(cl.change?.change ?? {})
+            .filter(([k, _]: [string, any]) => k !== 'id')
+            .map(([k, _]: [string, any], i) => `${k} = ${updatedValues[i]}`)
+            .join(', ')}
+          WHERE ${Object.entries(cl.change?.original ?? {})
+            // We need to add an special case for AMIs since we know the revolve string can be used and it will not match with the actual AMI assigned
+            .filter(([k, _]: [string, any]) => k !== 'ami' && k !== 'id')
+            .map(([k, _]: [string, any], i) => `${k} = ${conditionValues[i]}`)
+            .join(' AND ')};
+        `;
+        break;
+      default:
+        break;
+    }
+    if (query) queries.push(query);
+  }
+  return queries;
+}
+
+/** @internal */
+async function getValue(
+  tableName: string,
+  k: string,
+  v: any,
+  mbt: { [key: string]: ModuleInterface },
+  orm: TypeormWrapper,
```

Comment:
I think a bit of inline documentation and more clear variable naming could help here.

Reply:
Thank you for the feedback! I have added more comments and updated some variables' names. Let me know if it is a bit clearer now, or what else I could do to make it clearer